### ISSUE TITLE
Improvements on guide page interact-instance.html

### ISSUE THREAD

**Issue by mh-concordium** [Concordium/concordium.github.io#633]
**Description**
- It was not clear to me that passing parameters to methods was possible only with a file, I think it would be good to explicitly state that. I thought that it could be possible doing it directly with concordium-client
- It would be nice to have a link to where we explained error codes (-1, -2...)

**Comment by dg-concordium** [comment]
@abizjak I believe point 2 is related to #350, no?

**Comment by abizjak** [comment]
@dg-concordium Yes, partly. The other part is already covered in https://developer.concordium.software/en/mainnet/smart-contracts/guides/custom-errors.html 

The main problem is that it is hard to find unless you already know the right keywords. I am not sure how to solve that problem.
